From 20ae4425e9fb68bfaa92d67d51122279f37b0cca Mon Sep 17 00:00:00 2001
From: Matyas Selmeci <matyas@cs.wisc.edu>
Date: Mon, 6 Jul 2015 16:54:05 -0500
Subject: [PATCH 1/2] DERString -> ASN1String (bc1.47)

A class rename for the bouncycastle 1.47 api break from DERString to ASN1String.
---
 .../java/org/opensciencegrid/authz/xacml/common/X509CertUtil.java     | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/main/java/org/opensciencegrid/authz/xacml/common/X509CertUtil.java b/src/main/java/org/opensciencegrid/authz/xacml/common/X509CertUtil.java
index 6f953c4..a223dad 100644
--- a/src/main/java/org/opensciencegrid/authz/xacml/common/X509CertUtil.java
+++ b/src/main/java/org/opensciencegrid/authz/xacml/common/X509CertUtil.java
@@ -25,11 +25,11 @@ import org.gridforum.jgss.ExtendedGSSContext;
 import org.bouncycastle.asn1.x509.TBSCertificateStructure;
 import org.bouncycastle.asn1.x509.X509Name;
 import org.bouncycastle.asn1.ASN1Sequence;
 import org.bouncycastle.asn1.ASN1Set;
 import org.bouncycastle.asn1.DERObjectIdentifier;
-import org.bouncycastle.asn1.DERString;
+import org.bouncycastle.asn1.ASN1String;
 import org.glite.voms.*;
 import org.glite.voms.ac.AttributeCertificate;
 import org.glite.voms.ac.VOMSTrustStore;
 import org.glite.voms.ac.ACValidator;
 import org.glite.voms.ac.ACTrustStore;
@@ -518,11 +518,11 @@ attribute : /cms/uscms/Role=cmsprod/Capability=NULL
 		    buf.append(oid.getId());
 		} else {
 		    buf.append(sym);
 		}
 		buf.append('=');
-		buf.append( ((DERString)s.getObjectAt(1)).getString());
+		buf.append( ((ASN1String)s.getObjectAt(1)).getString());
 		if (ee.hasMoreElements()) {
 		    buf.append('+');
 		}
 	    }
 	  }
-- 
1.8.3.1

